Solutions/Proofpoint On demand(POD) Email Security/Parsers/ProofpointPOD.yaml

```diff
@@ -73,13 +73,78 @@ FunctionQuery: |
                     DstUserUpn,
                     NetworkDuration
     };
+    let ProofpointPOD_maillogV2_view = view () {
+        ProofpointPODMailLog_CL |
+         extend SmMsgid = tostring(parse_json(sm).msgid),
+            PpsCid = tostring(parse_json(pps).cid),
+            PpsAgent = tostring(parse_json(pps).agent),
+            Id = column_ifexists('id', ''),
+            SmNrcpts = tostring(parse_json(sm).nrcpts),
+            SmAuth = tostring(parse_json(sm).auth),
+            SmClass = tostring(parse_json(sm).class),
+            SmQid = tostring(parse_json(sm).qid),
+            MetadataOriginDataAgent = tostring(parse_json(metadata).origin.data.agent),
+            MetadataOriginDataCid = tostring(parse_json(metadata).origin.data.cid),
+            SmMailer = tostring(parse_json(sm).mailer),
+            SmDsn = tostring(parse_json(sm).dsn),
+            SmPri = tostring(parse_json(sm).pri),
+            SmXdelay = tostring(parse_json(sm).xdelay),
+            SmCtladdr = tostring(parse_json(sm).ctladdr),
+            EventUid = tostring(parse_json(sm).msgid), // need to verify
+            NetworkBytes = toreal(parse_json(sm).sizeBytes),
+            //TlsEstablished = parse_json(sm).tls_verify,
+            SrcNatIpAddr = tostring(parse_json(sm).relay),
+            ProcessName = tostring(parse_json(sm).daemon),
+            NetworkProtocol = tostring(parse_json(sm).proto),
+            SrcUserUpn = tostring(parse_json(sm).from),
```

Comment:
Please avoid using multiple parse_json for same fields like 'sm' is being parsed multiple times. Do parse_json once, it would become dynamic and then extract different fields from that dynamic field

Reply:
@vakohl , I have updated this, by creating variables and using them.